src/main/resources/templates/macros/report/embeddings.vm

```diff
@@ -1,6 +1,6 @@
 #macro(includeEmbeddings, $embeddings)
 
-#if ($embeddings)
+#if ($embeddings && $embeddings.size() > 0)
```

Comment:
in that case I wonder if second condition is not enough? Also maybe ``!$embeddings.isEmpty()``